components/list-item/README.md

```diff
@@ -7,8 +7,17 @@ ListItem
 ```
 <!-- STORY -->
 
+### Usage
 
 
+Simple
+```js
+<ListItem>List item example</ListItem>
```

Comment:
Might be nice to demonstrate usage with the asAnchor HOC, but not essential for this PR